packages/wallet/test/services/HandlesTracker.test.ts

```diff
@@ -26,39 +41,80 @@ const expectedHandleInfo = {
   // as AssetInfo is just some supplementary data
   resolvedAt: ledgerTip,
   supply: 1n
-} as HandleInfo;
+};
+
+const expectedHydratedHandleInfo: HandleInfo = {
+  ...expectedHandleInfo,
+  backgroundImage: Asset.Uri('ipfs://Q2de4Fg56tNHy82300000001'),
+  profilePic: undefined
+};
+
+const hydrateHandleSync = (handleInfo: HandleInfo) =>
+  of({
+    ...handleInfo,
+    backgroundImage: expectedHydratedHandleInfo?.backgroundImage,
+    profilePic: expectedHydratedHandleInfo?.profilePic
+  });
+
+const delaySyntax = `${RESOLVE_HANDLE_DELAY - 1}ms`;
 
 describe('createHandlesTracker', () => {
   it('matches utxo$ assets to given handlePolicyIds and adds context from assetInfo$ and tip$ observables', () => {
     createTestScheduler().run(({ hot, expectObservable }) => {
-      const utxo$ = hot(     '-ab-', { a: utxo, b: utxo2 });
-      const tip$ = hot(      'a-b-', { a: ledgerTip, b: ledgerTip2 });
+      const utxo$ = hot('     -ab-', { a: utxo, b: utxo2 });
```

Comment:
Best to avoid these types of formatting changes

Reply:
I think my local prettier was removing the empty spaces outside the quotes. I'll revert.

Reply:
fixed here: https://github.com/input-output-hk/cardano-js-sdk/pull/786/commits/8eb8916389fd57e318234f41ebb98ff52d28d63d.